test/api-call.ts

```diff
@@ -0,0 +1,27 @@
+const request = require('request');
+const fs = require('fs');
+
+//Requesting runs as a json and writing to file
+
+//build strings for paths/filename
+let d = new Date();
+let dateYMD = "" + d.getFullYear() + (d.getMonth()+1) + d.getDate();
+
+const folderpath = './test/jsonblob/';
+let fn = '-jak1.json';
+let filename = dateYMD + fn;
+
+
+let url = 'https://www.speedrun.com/api/v1/runs?game=xkdk4g1m&max=10';
+
+//make the request
+request(url, (err, res, body) => {
+    if (err) { return console.log(err); }
+    
+    //write body to filestream
+    let stream;
+    stream = fs.createWriteStream(folderpath + filename);
+    stream.write(body);
+
+});
```

Comment:
I'd recommend pulling in a dependency to make this easier https://www.npmjs.com/package/axios since `fetch` is still kinda new in terms of being included in nodejs by default.

You could turn the code into essentially two lines:
```ts
async function getRuns(gameId: string, runId: string) {
	const response = await axios.get(`https://www.speedrun.com/api/v1/$runId?game=$gameId&max=100`);
  console.log(response);
}
```